### PR TITLE
feat(730): we should add multiple language alternatives for seo

### LINKED ISSUE #730: feat: we should add multiple language alternatives for seo
## Define your request
Required to display results in google searches with multilingual support

https://developers.google.com/search/docs/specialty/international/localized-versions?hl=tr#methods-for-indicating-your-alternate-pages

** discord username: Hakan Angın#9836 **

## Define the alternatives you have in mind
_Not have_

## Extra context
_Not have_

### PR DESCRIPTION
**Pull request by khnbzkrt**
# Description
Added multiple language alternatives for google indexing

**discord username: Hakan Angın#9836 **

**closes #730 **

Please describe your changes. Also describe your aim and content. Do not forget to list the dependencies required caused by those changes.

## Things to check before creating a PR
- [ ] I have inspected my topic, checked.
- [ ] If it is a core feature, executed detailed tests.

## Creating PR rules
- [ ] PR must be created for an issue with approved tag. Otherwise PR will be rejected.
- [ ] Relevant issue number: The issue number related to PR must be attached to head of PR header, after prefix after prefix # must be attached in parenthesis. A header like this could be used "prefix(#issue_number): PR header" .
- [ ] A descriptive and understandable title: The PR title should clearly describe the nature and purpose of the changes. The PR title should be the first thing displayed when the PR is opened. And it should follow the semantic commit rules. For example, a title like "docs(#issueId): Add README.md" can be used.
- [ ] Related file selection: Only relevant files should be touched and no other files should be affected.
- [ ] Format and Lint Suitability : The code should be made in accordance with a certain format standard and examined according to the lint rules.
- [ ] Clean commit history: The commits where the changes are made should be clear and organized.
- [ ] Opening PR when job is completed: PR should be opened when job is completed and sent for review by other team members.

## Changes

- [ ] A new feature (a change that adds a new feature, not a breaking change)
- [ ] A new refactor (a change that is not a breaking change, that improves the readability or performance of the code)
- [ ] A breaking change
- [ ] Documentation change

# How were these changes tested?

Please describe the tests you did to test the changes you made. Please also specify your test configuration.

**Test Configuration**:

* Firmware version:
* Hardware:
* Toolchain:
* SDK:

### CODE REVIEW

**Comment by fdemir** [comment]
reproduce edilemiyor. vercel'e erisim yok. erisimi olan biri bakabilir.

File: pages/_error.tsx [hunk 1]
```diff
@@ -41,11 +40,7 @@ const CustomErrorPage: NextPage<CustomErrorPageProps> = ({ title, detail }) => {
   );
 };
 
-CustomErrorPage.getInitialProps = async (contextData) => {
-  return NextErrorComponent.getInitialProps(contextData);
-};
-
-export async function getStaticProps(context: any) {
+export async function getServerSideProps(context: any) {
```
Comment (cdagli) [inline pages/_error.tsx:43]: @usirin Could this be the fix for the lines that you commented out?

**Comment by oguzergul** [comment]
I have checked the preview link for metadata but it looks not okay. It is always Turkish. @khnbzkrt could you please check it?

**Comment by khnbzkrt** [comment]
I'm looking

**Comment by khnbzkrt** [comment]
@oguzergul this issue fixed you can continue. Thanks for your patience 👍🙏❤

File: components/base/HeadWithMeta/HeadWithMeta.tsx [hunk 2]
```diff
@@ -84,6 +85,16 @@ const HeadWithMeta = (props: IHeadWithMeta) => {
...
+      languageAlternates={[
+        {
+          hrefLang: SEO_LANG.HREFLANG_TR,
+          href: SEO_LANG.HREF_TR,
+        },
+        {
+          hrefLang: SEO_LANG.HREFLANG_EN,
+          href: SEO_LANG.HREF_EN,
+        },
+      ]}
```
Comment (cdagli) [inline components/base/HeadWithMeta/HeadWithMeta.tsx:97]: Bu option sunlari mi ekliyor head icerisine? 

image

Comment (khnbzkrt) [reply components/base/HeadWithMeta/HeadWithMeta.tsx:97]: Yesss 👍

**Review by fdemir — APPROVED** [review]

**Review by serhhan — APPROVED** [review]

**Pull request merged**